src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -359,8 +359,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
       _runtimeClassLoader
     })
 
-  // Set the current Java "context" class loader to this interpreter's class loader
-  def setContextClassLoader() = classLoader.setAsContext()
+  def setContextClassLoader() = ()
```

Comment:
I went too far; I'll back off the changes. Comment on another PR says sbt uses this.